examples/cluster/plot_digits_linkage.py

```diff
@@ -30,9 +30,7 @@
 
 from sklearn import manifold, datasets
 
-digits = datasets.load_digits(n_class=10)
-X = digits.data
-y = digits.target
+X, y = datasets.load_digits(n_class=10, return_X_y=True)
```

Comment:
n_class=10 seems redundant

Reply:
@qinhanmin2014 thanks, I removed the `n_class` parameter.